addons/skin.confluence/720p/MusicOSD.xml

```diff
@@ -332,8 +343,8 @@
 				<texturefocus>OSDRecordOffFO.png</texturefocus>
 				<texturenofocus>OSDRecordOffNF.png</texturenofocus>
 				<onclick>PlayerControl(record)</onclick>
-				<enable>Player.CanRecord</enable>
 				<animation effect="fade" start="100" end="50" time="75" condition="!Player.CanRecord">Conditional</animation>
```

Comment:
Sorry, leave the button 703 as it is and this is good to go.

Thanks for the fixes.